Sources/Sentry/SentryCrashWrapper.m

```diff
@@ -0,0 +1,21 @@
+#import "SentryCrashWrapper.h"
+#import "SentryCrash.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation SentryCrashWrapper
```

Comment:
L: on Android we use the naming NameOfTheClass`Adapter` for the same reason, wdyt? so we'd keep the same naming across both SDKs

Reply:
Good point, I like `Adapter` more. I think it is more precise in this situation here.